run/helloworld/go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/GoogleCloudPlatform/golang-samples/run/helloworld
 
-go 1.13
+go 1.19
```

Comment:
Is there motivation for this besides just to bump it up? This really only affects code being allow to use new language features. Also if this is going to get bumped we should be sure to bump the appropriate Dockerfile to match.

Reply:
After talking offline this seems fine, but I think we should bump the dockerfile too stil.

Reply:
Sounds good! Added commit to bump the Dockerfile.